includes/Dokan.php

```
@@ -163,4 +166,29 @@ public function add_dokan_settings( $settings_fields ) {
+     *
+     * @since 1.3.3
+     *
+     * @param bool     $should_exclude Whether to exclude the payment.
+     * @param WC_Order $order          Order object.
+     * @param int      $order_id       Order ID.
+     * @param string   $new_status     New order status.
+     * @param bool     $exclude_cod    Whether exclude COD option is enabled.
+     *
+     * @return bool True if payment should be excluded, false otherwise.
+     */
+    public function exclude_wepos_cash_payment( $should_exclude, $order, $order_id, $new_status, $exclude_cod ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$order_id'. (undefined)

(UnusedFormalParameter)